docs/src/componentDocs/ToolbarMenu/examples/ToolbarMenuWithIconExample.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import { ToolbarMenu } from '@brightlayer-ui/react-components';
+import { Home } from '@mui/icons-material';
+
+export const ToolbarMenuWithIconExample = (): JSX.Element => (
+    <Box>
+        <ToolbarMenu
+            sx={{ maxWidth: 'inherit' }}
```

Comment:
Do we need sx prop in code snippet block?

Reply:
Yes we need it in the example, but we can remove it from snippet